docs/design/w3c/w3c-representation.md

```diff
@@ -0,0 +1,457 @@
+## Design for W3C representation of AnonCreds credentials and presentation
+
+Currently `anoncreds-rs` library only provides support for Indy styled AnonCreds credentials matching to
+the [specification](https://hyperledger.github.io/anoncreds-spec/).
+
+This design document proposes extending of `anoncreds-rs` library to add support for AnonCreds W3C representation of
+verifiable credential and presentation described in the [document]().
+
+### Goals and ideas
+
+* Use Indy styled credentials to generate W3C AnonCreds credentials and presentations
+    * Credentials conversion:
+        * Convert Indy styled AnonCreds credentials into W3C form
+        * Convert W3C styled AnonCreds credentials into Indy form
+    * Presentation conversion (Optional):
+        * Convert Indy styled AnonCreds presentation into W3C form
+        * Convert W3C styled AnonCreds presentation into Indy form
+* Extend W3C credentials:
```

Comment:
As noted earlier — careful with the wording here, as the W3C AnonCreds format credentials are Data Integrity Proofs.